workflows/dags/literature/hep_create_dag.py

```diff
@@ -199,6 +199,94 @@ def count_reference_coreness(**context):
                 overwrite=True,
             )
 
+        @task(trigger_rule=TriggerRule.NONE_FAILED_MIN_ONE_SUCCESS)
+        def normalize_journal_titles(**context):
+            """
+            Normalize the journal titles in workflow data.
+
+            This function:
+            - Collects all journal titles from `publication_info` and
+                reference `publication_info`.
+            - Calls the Inspire API to normalize them.
+            - Updates each entry with the normalized title, journal record, and
+                categories.
+
+            Args:
+                context: Airflow context containing `params.workflow_id`.
+
+            Returns:
+                None
+            """
+            s3_workflow_id = context["params"]["workflow_id"]
+            workflow_data = read_object(s3_hook, bucket_name, s3_workflow_id)
+            data = workflow_data.get("data", {})
+
+            titles_to_normalize = get_value(
+                data, "publication_info.journal_title", []
+            ) + get_value(
+                data, "references.reference.publication_info.journal_title", []
+            )
+            if not titles_to_normalize:
+                return
+
+            response = inspire_http_hook.call_api(
+                endpoint="api/curation/literature/normalize-journal-titles",
+                method="GET",
+                json={"journal_titles_list": titles_to_normalize},
+            )
+            response.raise_for_status()
+            normalized_titles_map = get_value(
+                response.json(), "normalized_journal_titles", []
+            )
+            normalized_references_map = get_value(
+                response.json(), "normalized_journal_references", []
+            )
+            normalized_categories_map = get_value(
+                response.json(), "normalized_journal_categories", []
+            )
+
+            def process_entries(entries, title_getter):
```

Comment:
lets extract this to a utility func, similar to what we do with guess_coreness.py etc